modules/gitops-release-notes-1-12-0.adoc

```diff
@@ -0,0 +1,143 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-0_{context}"]
+= Release notes for {gitops-title} 1.12.0
+
+{gitops-title} 1.12.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-12.0_{context}"]
+== Errata updates
+
+[id="gitops-1-12-0-security-update-advisory_{context}"]
+=== RHSA-xxxx:nnnn - {gitops-title} 1.12.0 security update advisory
+
+Issued: 2024-03-14
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2023:6220[RHSA-xxxx:nnnn]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+*The Title and errata number will change after the errata state moves from PUSH STATE to SHIPPED LIVE. The current entry is simply a placeholder at the moment*.
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* With this update, the Argo CD CLI is supported and accessible as a productized and is an easier method for installation because it uses an RPM binary versus XM. By using the Argo CD CLI tool, you can complete the following tasks:
+** Manage {gitops-title} from a terminal.
+** Manage ArgoCD resources, such as `Applications`, `ApplicationSets`, `AppProjects`, `User accounts`, and `GPG keys` from a client terminal. link:https://issues.redhat.com/browse/GITOPS-3389[GITOPS-3389]
++
+[NOTE]
+====
+The `argocd` executable binary file is included in the archive and RPM formats.
+====
++
+[WARNING]
+==== 	
+The Argo CD CLI tool is a Technology Preview feature.
+====
+
+* With this update, `NotificationsConfiguration` custom resource (CR) is now supported. Before this update, you could update the `argocd-notifications-cm` ConfigMap to manage their templates, triggers, services and subscriptions. After this update, you cannot modify the `argocd-notifications-cm` ConfigMap. A new Custom Resource definition `NotificationsConfiguration` is introduced to manage the `argocd-notifications-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
```

Comment:
```suggestion
* With this update, `NotificationsConfiguration` custom resource (CR) is now supported. Before this update, you had to update the `argocd-notifications-cm` ConfigMap to manage templates, triggers, services and subscriptions. With this release, you cannot modify the `argocd-notifications-cm` ConfigMap. A new Custom Resource definition `NotificationsConfiguration` is introduced to manage the `argocd-notifications-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-4130[GITOPS-4130]
```

Reply:
A customer can update `NotificationsConfiguration`?

Reply:
Yes, as per the information provided in the release notes document. Fixed.